pacstall

```diff
@@ -205,6 +205,26 @@ fi
 
 source "$STGDIR/scripts/error_log.sh"
 
+# Separate grouped short options
+argument_list=()
+
+for i in "${@}"; do
+	# Just add argument if doesn't start with a single hyphen.
+	if ! [[ "${i}" =~ ^-[^-] ]]; then
+		argument_list+=("${i}")
+		continue
+	fi
+
+	# Add all arguments to the list of arguments.
+	# We remove the '-' prefix as we'll add it later.
+	for j in $(echo "${i}" | sed 's|^-||' | sed 's|[[:upper:]]| &|g'); do
+		argument_list+=("-${j}")
+	done
+done
+
```

Comment:
If the new arguments list has a `-P` you should move it to the start so that the prompts are disabled properly

Reply:
Done